packages/nimble-components/src/table/specs/table-column-specs/table-column-text-field.md

```diff
@@ -0,0 +1,175 @@
+# TableColumnTextField
+
+## Overview
+
+The `nimble-table-column-text-field` is a component that defines how to render a cell for that column in a `nimble-table` as text.
+
+### Background
+
+[Table Column API](../table-columns-hld.md)
+
+[Table Spec](../README.md)
+
+[Visual Design Spec](https://xd.adobe.com/view/5b476816-dad1-4671-b20a-efe796631c72-0e14/specs/)
+
+### Non-goals
+
+-   Defining an API that supports editable text. Once we need editable text we will determine if it makes sense to modify this component or introduce a new column type.
+
+### Features
+
+-   Offers a `placeholder` attribute that allows a user to define what text to render when no value is provided by the `nimble-table` data.
+
+---
+
+## Design
+
+Below is an example of how the `nimble-table-column-text-field` would be used within a `nimble-table`:
+
+```HTML
+<nimble-table>
+    <nimble-table-column-text-field value-key="firstName" placeholder="No data"></nimble-table-column-text-field>
+    <nimble-table-column-text-field value-key="lastName" placeholder="No data"></nimble-table-column-text-field>
+</nimble-table>
+```
+
+### API
+
+_Component Name_
+
+-   `nimble-table-column-text-field`
+
+_*Props/Attrs*_
+
+-   `value-key`: string
```

Comment:
Do you think this be called `value-field-name`? That would be more consistent with our type terminology, but it would also be more verbose.

Reply:
Any opinion @jattasNI? I lean towards `value-key` for brevity.

Reply:
Didn't we align on Records and Fields as the terminology for data?

I'd vote for `field-name` aligns with the attribute [`id-field-name` being proposed](https://github.com/ni/nimble/pull/937/files#diff-5d4eacdc37de4f42479b3c84514fdd15c02fd432ac7341cf2b4c68298723cc75R137).

Reply:
Updated.